typescript/cli/src/config/chain.ts

```diff
@@ -94,7 +95,9 @@ export async function createChainConfig({
 
   const parseResult = ChainMetadataSchema.safeParse(metadata);
   if (parseResult.success) {
-    logGreen(`Chain config is valid, writing to registry`);
+    logGreen(`Chain config is valid, writing to registry:`);
+    const metadataYaml = yamlStringify(metadata, null, 2);
+    log(indentYamlOrJson(metadataYaml, 4));
```

Comment:
think you might want `stringifyObject` in `@hyperlane-xyz/utils`

```suggestion
    log(stringifyObject(metadata, 'yaml', 4));
```

Reply:
hmm `stringifyObject(metadata, 'yaml', 4)` seems to ==  `yamlStringify(metadata, null, 2)`

and then the `4` in `indentYamlOrJson(metadataYaml, 4)` is to indent the entire block to stdout (not the internal tabs) like such:
```
➜  cli git:(06-24-chore_cli_display_new_chain_config_after_registry_init) hl registry init --registry $HOME/workplace/Hyperlane/hyperlane-registry
Hyperlane CLI

Creating a new chain config
? Enter http or https rpc url: https://alpha-arb-net.rpc.caldera.xyz/http
? Enter chain name (one word, lower case) alpha
? Enter chain display name Alpha
? Detected chain id as 75904 from JSON RPC provider, is this correct? yes
? Do you want to set block or gas properties for this chain config no
? Do you want to set native token properties for this chain config (defaults to ETH) no
Chain config is valid, writing to registry:
    name: alpha
    displayName: Alpha
    chainId: 75904
    domainId: 75904
    protocol: ethereum
    rpcUrls:
      - http: https://alpha-arb-net.rpc.caldera.xyz/http
    nativeToken:
      symbol: ETH
      name: Ether
      decimals: 18
    
Now updating chain alpha at filesystem registry at /Users/nbayindirli/workplace/Hyperlane/hyperlane-registry
Done updating chain alpha at filesystem registry
Now updating chain alpha at filesystem registry at /Users/nbayindirli/.hyperlane
Done updating chain alpha at filesystem registry
```